### PR TITLE
Docs: Create your component section

### PR DESCRIPTION
**Pull request by NRodriguezcuellar**
Added a small section on how to customize RSK with your own components using react-overridable

### CODE REVIEW

**Review by Pineirin — COMMENTED** [review]
Nice to have this documented with examples.
Still, if possible, I think it would be easier to understand if it was more clear when we are overriding something and when we are making something "overridable".

File: docs/docs/components/react_search_kit.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ See the [complete guide](main_concepts.md) for detailed information.
 
 - **appName** `string` _optional_
 
-  A name identifier to distinguish uniquely the application. Useful if multiple ReactSearchKit apps are loaded in the same page. Default `RSK`.
+  A name identifier to distinguish uniquely the application. Useful if multiple ReactSearchKit apps are loaded in the same page, if overriding components the overridable-id will be prefixed by this appName. Default ``.
```
Comment (ntarocco) [inline docs/docs/components/react_search_kit.md:47]:
```suggestion
  An name to uniquely identify the application. Useful when enabling the `event listener` (read below) and when multiple ReactSearchKit apps are loaded in the same page. The `overridable-id` of each component will be prefixed by this `appName`. Default is empty string (no namespacing).
```

File: docs/docs/create_your_component.md [hunk 2]
```diff
@@ -3,4 +3,30 @@ id: create-your-component
 title: Create Your Component
 ---
 
-To be documented
+To allow for customization we use [React overridable](https://github.com/indico/react-overridable).
+
+By passing a map with IDs and components we can override the default components with our own custom components.
```
Comment (ntarocco) [inline docs/docs/create_your_component.md:8]: Can you please add a sentence to explain how to find the `IDs` to override?

File: docs/docs/ui_customisation.md [hunk 3]
```diff
@@ -6,3 +6,44 @@ title: Customise The Look And Feel
...
+};
+
+<OverridableContext.Provider value={overriddenComponents}>
+          <ReactSearchKit/>
+</OverridableContext.Provider>
+
+```
+ 
+The ID used for overriding a component can be found by searching the component in the RSK [library](https://github.com/inveniosoftware/react-searchkit/tree/master/src/lib/components)
+and looking at the ID passed to the `Overridable` component. The ID is build using a function called `buildUID` which allows us to namespace our IDs if needed.
```
Comment (ntarocco) [inline docs/docs/ui_customisation.md:37]:
```suggestion
and looking at the ID passed to the `Overridable` component. The ID is built using a function called `buildUID` which allows us to namespace our IDs if needed.
```

**Review by ntarocco — APPROVED** [review]

**Review by zzacharo — APPROVED** [review]

**Pull request merged**